oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxHWDiskStore.java

```diff
@@ -272,8 +272,8 @@ private static List<HWDiskStore> getDisks(LinuxHWDiskStore storeToUpdate) {
         }
         // Iterate the list and make the partitions unmodifiable
         for (HWDiskStore hwds : result) {
-            ((LinuxHWDiskStore) hwds).partitionList = Collections.unmodifiableList(hwds.getPartitions().stream()
```

Comment:
hmmm CI throwing errors not sure why, this is introduced since [10](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/stream/Collectors.html#toUnmodifiableList())

Reply:
>  this is introduced since [10](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/stream/Collectors.html#toUnmodifiableList())

We still compile with compatibility for 8, at least until 31 Mar 2025.